### ISSUE TITLE
Convert NSSet to Set

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#141]
Swift 2.0 has a native `Set`, which is bridged with `NSSet`.

Example:

```
+(NSSet *)keyPathsForValuesAffectingValueForKey:(NSString *)key {
     return nil;
}
```

Current:

```
	func keyPathsForValuesAffectingValueForKey(key: String) -> NSSet {
		return nil
	}
```

Should be:

```
	func keyPathsForValuesAffectingValueForKey(key: String) -> Set<String> {
		return nil
	}
```

**Comment by drkameleon** [comment]
Question:

If you have `NSSet<NSString *> *` (generics) as return type, it can sure be converted to `Set<String>`.

What if there's no generics, and it's just `NSSet*` (like in your example)? How is it to guess the type? Could it also be just `Set`? (in the Swift output, I mean...)

**Comment by charlieMonroe** [comment]
Oh, then it's my bad, I though you already have :)

If no generics are provided, `AnyObject` should be assumed - see my other issue about arrays and dictionaries.

**Comment by drkameleon** [comment]
You mean like `Set<AnyObject>`?

**Comment by charlieMonroe** [comment]
Yes, exactly.

**Comment by drkameleon** [comment]
Noted. ;-)

------

P.S. 

a) thanks a lot (one more time) for all the bug reporting and the suggestions (keep it coming)
b) a... self-explanatory sneak-peak for you of what's coming, directly from the new "under construction" wiki: https://github.com/drkameleon/iswift/wiki/A-quick-(visual)-Walk-Through (yes, I'm talking about the "File Manager" - for now, it is what it is: a way to have more than one files open. Nothing more. Although it's also a first step towards something more... project-oriented ;-) )

**Comment by charlieMonroe** [comment]
Looks great, looking forward to it!

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.3.
